input/fsh/SD_TumorSize.fsh

```diff
@@ -87,32 +87,20 @@ Description:  "Identifies a tumor that has not been removed from the body. Whene
 Profile: BodyStructureIdentifier
 Parent: Identifier
 Id: mcode-body-structure-identifier
-Title: "Body Structure Identifier Data Type Profile"
+Title: "Body Structure Identifier Profile"
 Description: "An identifier that designates a body structure such as a tumor and can be used for tracking the structure over time."
 * ^extension[FMM].valueInteger = 3
 * type = http://hl7.org/fhir/resource-types#BodyStructure
 * value 1..1
 * use = #usual
 
 Profile: TumorSpecimen
-Parent: Specimen
+Parent: PatientSpecimen
```

Comment:
I don't understand why you need the TumorSpecimen profile.
If it is only for the FMM that seems gratuitous.
The profile is only differentiated by meta.profile, if included.

Reply:
You have a good point. It could be removed, and just document that the type = #TUMOR.